float32[3] cameraPos
---
bool success
